src/Report.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <thread>
+
+#include "KafkaW.h"
+#include "Status.hpp"
+#include "StatusWriter.hpp"
+#include "logger.h"
+
+namespace KafkaW {
+class ProducerTopic;
```

Comment:
Why is the forward declaration needed if you `#include "KafkaW.h"`?

Reply:
:white_check_mark: